src/maud/cli.py

```diff
@@ -87,6 +87,58 @@ def sample_command(data_path, output_dir):
     click.echo(sample(data_path, output_dir))
 
 
+def ppc(samples_path, ppc_path, output_dir):
```

Comment:
I think we should avoid the name `ppc` entirely as it could confuse people

Maybe `generate_predictions` for the function name and `oos_path` for the path